src/node_file.cc

```diff
@@ -464,7 +465,7 @@ void FillStatsArray(double* fields, const uv_stat_t* s) {
   // Dates.
 #define X(idx, name)                                                          \
   fields[idx] = (static_cast<double>(s->st_##name.tv_sec) * 1000) +           \
-                (static_cast<double>(s->st_##name.tv_nsec / 1000000));        \
+                round(static_cast<double>(s->st_##name.tv_nsec) / 1000000);   \
```

Comment:
I'm not a C++ maven but maybe
`(static_cast<double>((s->st_##name.tv_nsec + 500000) / 1000000))`
Would be faster

Also If you're already here could you `#define` the magic numbers please?